src/ontology/mondo-edit.obo

```diff
@@ -358801,9 +358776,7 @@ xref: ICD10CM:Q39.2 {source="Orphanet:454750", source="Orphanet:454750/e"}
 xref: NCIT:C35080 {source="MONDO:equivalentTo"}
 xref: Orphanet:454750 {source="MONDO:equivalentTo"}
 xref: UMLS:C0040588 {source="Orphanet:454750", source="MONDO:notFoundInDiseaseSubset"}
-is_a: MONDO:0015207 {source="Orphanet:454750"} ! non-syndromic esophageal malformation
-is_a: MONDO:0015221 {source="Orphanet:454750"} ! non-syndromic respiratory or mediastinal malformation
-is_a: MONDO:0015930 {source="Orphanet:454750"} ! respiratory malformation
+is_a: MONDO:0003749 {source="https://orcid.org/0000-0001-5208-3432"} ! esophageal disorder
```

Comment:
Could this also be a 'tracheal disorder' MONDO:0002567?

Reply:
I added this